docs/source/contributing.rst

```diff
@@ -0,0 +1,71 @@
+Undebt_: Contributing
+==========================
+
+.. _Undebt: index.html
+.. default-role:: code
+
+Getting Started
+---------------
+
+Undebt's development is taking place on Github, so please go ahead and `fork`_ the repository if you want to begin contributing.
+
+.. _`fork`: https://github.com/Yelp/undebt#fork-destination-box
+
+You'll then want to get a local copy of the code base:
+
+.. code-block:: bash
+
+    git clone git@github.com:<your-username>/undebt.git
+
+Getting Setup
```

Comment:
We have `make` targets for each of these steps.

- `make venv`
  - sets up virtualenv
  - worth adding a reminder to source `venv/bin/activate`
- `make dev`
  - all of the above + requirements-dev.txt
- `make test`
  - all of the above + runs tests
  - worth noting that once `make dev` has been run and you are in the virtualenv, you can use `tox -e <my-env>` to run tests for just one python, etc